contracts/0.4.24/Lido.sol

```diff
@@ -651,6 +673,156 @@ contract Lido is ILido, StETH, AragonApp {
         return EL_REWARDS_VAULT_POSITION.getStorageAddress();
     }
 
+    /**
+     * @dev updates beacon state and calculate rewards base (OUTDATED)
+     */
+    function _processAccounting(
+        // CL values
+        uint256 _beaconValidators,
+        uint256 _beaconBalance
+    ) internal returns (uint256 appearedValidators) {
+        uint256 depositedValidators = DEPOSITED_VALIDATORS_POSITION.getStorageUint256();
+        require(_beaconValidators <= depositedValidators, "REPORTED_MORE_DEPOSITED");
+
+        uint256 beaconValidators = BEACON_VALIDATORS_POSITION.getStorageUint256();
+        require(_beaconValidators >= beaconValidators, "REPORTED_LESS_VALIDATORS");
+
+        // Save the current beacon balance and validators to
+        // calculate rewards on the next push
+
+        BEACON_BALANCE_POSITION.setStorageUint256(_beaconBalance);
+
+        if (_beaconValidators > beaconValidators) {
+            BEACON_VALIDATORS_POSITION.setStorageUint256(_beaconValidators);
+        }
+
+        return _beaconValidators.sub(beaconValidators);
+    }
+
+    /**
+     * @dev move funds between ELRewardsVault, deposit and withdrawal buffers. Updates buffered counters respectively
+     */
+    function _processFundsMoving(
```

Comment:
`_processBufferedEtherDistribution`

Reply:
Can't agree here. It does not distribute buffered ether exactly. It also collects ELRewards and excess withdrawals. So it is not precise name for the method.

Reply:
Let's find a better one though. `FundsMoving` sounds too weird.